modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -94,6 +94,10 @@ def email
       veteran_contact_info['emailAddressText']
     end
 
+    def email_v2
```

Comment:
I think i'd prefer if we update the `email` method to:
```ruby
veteran_contact_info['email'].presence || veteran_contact_info['emailAddressText']
```

Reply:
agreed